bin/boot

```diff
@@ -25,6 +25,36 @@ function login  {
 }
 
 function start {
+    if [ "$ADMIN_PASSWORD" ]; then
+        sed -i "s/_ADMIN_PASSWORD_/$ADMIN_PASSWORD/g" /opt/odoo/etc/odoo.conf
+    fi
+
+    if [ "$DB_USER" ]; then
+        sed -i "s/_DB_USER_/$DB_USER/g" /opt/odoo/etc/odoo.conf
+    fi
+
+    if [ "$DB_PASSWORD" ]; then
+        sed -i "s/_DB_PASSWORD_/$DB_PASSWORD/g" /opt/odoo/etc/odoo.conf
+    fi
+
+    if [ "$ADDONS_REPO" ]; then
+        echo "Setting up SSH key"
+        cp -R /mnt/ssh /root/.ssh
+        chown -R $USER:$USER /root/.ssh
+        ssh_file="/root/.ssh/id_rsa"
+        if [ -f "$ssh_file" ]; then
+            chmod 600 /root/.ssh/id_rsa
+        fi
+        ssh-keyscan github.com  >> /root/.ssh/known_hosts
+        echo "Downloading additional addons..."
```

Comment:
@elicoidal @noahzaozao Sorry this conversation was hidden, I just saw it. IMO, no need for timestamps. We already have them in rsyslog. But I'm not against the idea, for instance if we want to see how long each step is taking and we use d-c logs instead of using rsyslog